### PR TITLE
UML-783 Updated panel title with LPA type

### PR DESCRIPTION
**Pull request by nickjones-moj**
## Purpose
Added LPA type to the panel on the LPA summary

Fixes UML-783

## Checklist

* [x] I have performed a self-review of my own code
* [ ] I have added relevant logging with appropriate levels to my code
* [ ] I have updated documentation (Confluence/GitHub wiki/tech debt doc) where relevant
* [ ] I have added tests to prove my work
* [x] The product team have tested these changes

### CODE REVIEW

File: service-front/app/features/context/UI/ViewerContext.php [hunk 1]
```diff
@@ -422,23 +422,4 @@ public function iWantToSeeAnOptionToCheckAnotherLPA()
...
-    public function iWaitedTooLongToEnterTheShareCode()
-    {
-        $this->ui->getSession()->setCookie('session', null);
-    }
-
-    /**
-     * @Then /^I have an error message informing me to try again\.$/
-     */
-    public function iHaveAnErrorMessageInformingMeToTryAgain()
-    {
```
Comment (williamfalconeruk) [inline service-front/app/features/context/UI/ViewerContext.php:438]: and this?

File: service-front/app/features/context/UI/ViewerContext.php [hunk 2]
```diff
@@ -422,23 +422,4 @@ public function iWantToSeeAnOptionToCheckAnotherLPA()
         $this->ui->clickLink('I want to check another LPA');
         $this->iGiveAValidLPAShareCode();
     }
-
-    /**
-     * @Given /^I waited too long to enter the share code$/
-     */
-    public function iWaitedTooLongToEnterTheShareCode()
```
Comment (williamfalconeruk) [inline service-front/app/features/context/UI/ViewerContext.php:429]: how come this was removed?

File: service-front/app/features/context/UI/ViewerContext.php (hunk 1 above)
Comment (williamfalconeruk) [reply service-front/app/features/context/UI/ViewerContext.php:438]: might've been a merge issue here - could you reinstate these lines?

File: service-front/app/features/context/UI/ViewerContext.php (hunk 2 above)
Comment (williamfalconeruk) [reply service-front/app/features/context/UI/ViewerContext.php:429]: this is likely a merge issue - can you reinstate this function?

**Comment by nickjones-moj** [comment]
> a couple of the test steps got wiped out (probably inadvertently) even after merging. this will need to be fixed.

Thanks, not sure what happen there. Now added back in.

**Pull request merged**